src/org/parosproxy/paros/core/proxy/ProxyThread.java

```diff
@@ -321,14 +325,33 @@ private static void setErrorResponse(HttpMessage msg, String responseStatus, Exc
     private static void setErrorResponse(HttpMessage msg, String responseStatus, Exception cause, String errorType)
             throws HttpMalformedHeaderException {
         StringBuilder strBuilder = new StringBuilder();
-        strBuilder.append(errorType)
-                .append(" [")
-                .append(cause.getClass().getName())
-                .append("]: ")
-                .append(cause.getLocalizedMessage())
-                .append("\n\nStack Trace:\n");
-        for (String stackTraceFrame : ExceptionUtils.getRootCauseStackTrace(cause)) {
-            strBuilder.append(stackTraceFrame).append('\n');
+		
+        if (cause instanceof SSLException) {
+            strBuilder.append(Constant.messages.getString("network.ssl.error.connect"));
+            strBuilder.append(msg.getRequestHeader().getURI().toString()).append('\n');
+            strBuilder.append(Constant.messages.getString("network.ssl.error.exception") + cause.getMessage())

```

Comment:
- Ideally it should also `append` the message of the exception.
 - It could use `ExceptionUtils.getRootCauseMessage(Throwable)` to obtain the message of the exception.